src/main/java/org/gradle/profiler/mutations/ApplyProjectDependencyChangeMutator.java

```diff
@@ -0,0 +1,35 @@
+package org.gradle.profiler.mutations;
+
+import org.gradle.profiler.BuildContext;
+import org.gradle.profiler.BuildInvoker;
+import org.gradle.profiler.GradleBuildInvoker;
+import org.gradle.profiler.mutations.support.ProjectCombinations;
+
+import java.io.File;
+import java.util.Set;
+
+public class ApplyProjectDependencyChangeMutator extends AbstractFileChangeMutator {
```

Comment:
It would be good to add some Javadoc here describing how this relates to `ApplyProjectDependencyChangeSetupMutator`. I wonder if the two could be a single mutator, or one delegating to the other or something, to capture the relationship?

Reply:
Merged these two mutators into one as suggested. Also added some javadoc what it does